DESCRIPTION

```diff
@@ -20,16 +20,23 @@ URL: https://github.com/r-lib/gh#readme
 BugReports: https://github.com/r-lib/gh/issues
 Imports:
     cli (>= 2.0.1),
+    curl,
```

Comment:
Temporary. If `set_github_pat2()` stays here, we can do token validation without a direct use of curl, i.e. using only gh functions.

Reply:
we can use curl, no problem.